### PR TITLE
Add Ctrl+F5 as a shortcut for reconnect()

### PR DESCRIPTION
**Pull request by mammothb**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description
Added a keyboard shortcut (Ctrl+F5) for "Reconnect"

### CODE REVIEW

**Review by zneix — CHANGES_REQUESTED** [review]
Please make sure to consistently reflect changes to keybinding in `Settings -> Keybindings`. Source for that page can be found in `KeyboardSettingsPage.cpp`.

**Comment by mammothb** [comment]
Do you mean this [line](https://github.com/Chatterino/chatterino2/blob/b98a0df73ba0e82527f360ae3948f0ac8831d919/src/widgets/settingspages/KeyboardSettingsPage.cpp#L69)? Or do you mean I should change the label to something else?

**Comment by zneix** [comment]
> Do you mean this [line](https://github.com/Chatterino/chatterino2/blob/b98a0df73ba0e82527f360ae3948f0ac8831d919/src/widgets/settingspages/KeyboardSettingsPage.cpp#L69)? 

Oh right, I totally missed that file in there. Apologies for the confusion!



> Or do you mean I should change the label to something else?

If I were to change anything, maybe `Reconnect current channel` would fit a bit better?

**Comment by mammothb** [comment]
> If I were to change anything, maybe `Reconnect current channel` would fit a bit better?

The `reconnect()` function will cause all channels to reconnect so I'm not sure if specifying "current channel" will be an accurate description of the functionality.

**Comment by zneix** [comment]
Oh yes, I believe you're right I just checkedthe source deeper and I mixed that up with regular reconnection. Once again I am sorry for messing up ;/

**Review by zneix — APPROVED** [review]

**Review by pajlada — COMMENTED** [review]
I am hesitant towards adding a hotkey for something this disruptive when the reason for its existence is a bug (Chatterino sometimes not reconnecting on its own). Is this an action you have to take often?

**Comment by mammothb** [comment]
It occurs irregularly but I had to reconnect a couple of times today, which prompted this modification. And as some people I talked to would rather get it from the official build than a custom build from me, I decided to open this PR.

**Comment by alazymeme** [comment]
I think it's a nice addition. I won't personally use it, but I see it being quite useful if I do ever need it.

**Review by pajlada — APPROVED** [review]
Approved under the condition that this will not stay forever - will be removed in a future patch soon :tm:

File: src/widgets/splits/Split.cpp [hunk 1]
```diff
@@ -118,6 +118,9 @@ Split::Split(QWidget *parent)
     // F5: reload emotes
     createShortcut(this, "F5", &Split::reloadChannelAndSubscriberEmotes);
 
+    // CTRL+F5: reconnect
+    createShortcut(this, "CTRL+F5", &Split::reconnect);
```
Comment (23rd) [inline src/widgets/splits/Split.cpp:122]: Why not just use a lambda?

**Pull request merged**